joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -428,25 +318,28 @@ TEST_P(TrajectoryControllerTestParameterized, correct_initialization_using_param
     EXPECT_LE(0.0, joint_eff_[2]);
   }
 
-  // cleanup
-  state = traj_controller_->get_node()->cleanup();
-  ASSERT_EQ(State::PRIMARY_STATE_UNCONFIGURED, state.id());
+  // reactivated
+  // wait so controller process the third point when reactivated
+  std::this_thread::sleep_for(std::chrono::milliseconds(3000));
+  // TODO(anyone) test copied from ROS 1: it fails now!
```

Comment:
I merged the old reactivation test into that one, should the old trajectory really be processed after reactivation? (it doesn't)

Reply:
It shouldn't imo and we should have a test to ensure it

Reply:
yes, I agree. Would be strange if it does. Writing the test for that is easy, because this one fails already :D